src/components/Registration/Registration.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC, FormEvent } from 'react';
+import Form from '../Form/Form';
+import DaysAndTime from '../DaysAndTime/DaysAndTime';
+import { RegistrationProps } from '../../interfaces/interfaces'
+
+
+
+const Registration: FC<RegistrationProps> = (props) => {
+  const { name, setName, username, setUsername, email, setEmail,
+    page, setPage, make, setMake, model, setModel, year, setYear, origin, 
+    setOrigin, destination, setDestination, departTime, setDepartTime, days, setDays,
+    password, setPassword, bio, setBio } = props;
+
+  const progress = {
+    transform: `scaleX(.${page * 20})`
+  };
+
+  const handleSubmit = (e:FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    setPage(page + 1)
+  }
+
+  return(
+    <form onSubmit={e => handleSubmit(e)} className="registration">
```

Comment:
Would we want to use router here or does conditional rendering make more sense?

Reply:
Yeah maybe there's a chance to refactor here!